docs/en/sql_reference/statements/alter.md

```diff
@@ -500,4 +500,92 @@ A mutation query returns immediately after the mutation entry is added (in case
 
 Entries for finished mutations are not deleted right away (the number of preserved entries is determined by the `finished_mutations_to_keep` storage engine parameter). Older mutation entries are deleted.
 
+## ALTER USER {#alter-user-statement}
+
+Changes ClickHouse user accounts.
+
+### Syntax {#alter-user-syntax}
+
+``` sql
+ALTER USER [IF EXISTS] name [ON CLUSTER]
```

Comment:
`ON CLUSTER 'cluster_name'`, not just `ON CLUSTER`
The same thing for all `ON CLUSTER`.